lib/dataset/query.js

```diff
@@ -338,6 +338,50 @@ define({
             return this.mergeOptions({distinct: args});
         },
 
+        /*   {
+         *     division: db.from('division').one();
+         *   }
+         *
+         */
+        eager: function(includeDatasets, model) {
+            this._eager = includeDatasets;
```

Comment:
The reason we do that is because we follow a builder pattern so this shouldnt modify the original instance instead return a new version with the eager method set.